build/create_binaries/README.md

```diff
@@ -0,0 +1,14 @@
+# Create Binaries
+
+
+Build the app with:
+
+```
+julia --project create_app.jl
+```
+
+<!-- Build the shared library with:
```

Comment:
I know this is a plan for later, but we could just remove these commented lines for now.

Reply:
Done!